code/game/machinery/poolcontroller.dm

```diff
@@ -100,15 +100,15 @@
 
 		if(drownee.stat == DEAD)	//Dead spacemen don't drown more
 			return
-		if(drownee.losebreath > 20)	//You've probably got bigger problems than drowning at this point, so we won't add to it until you get that under control.
+		if(drownee.lose_breath > 20)	//You've probably got bigger problems than drowning at this point, so we won't add to it until you get that under control.
```

Comment:
why arbitrarily adjust this from `losebreath` to `lose_breath` Would rather not mob vars be touched unless there's an exceptionally good reason for them to be touched.